x/ibc/20-transfer/keeper/keeper.go

```diff
@@ -80,3 +80,13 @@ func (k Keeper) ChanCloseInit(ctx sdk.Context, portID, channelID string) error {
 func (k Keeper) TimeoutExecuted(ctx sdk.Context, packet channelexported.PacketI) error {
 	return k.channelKeeper.TimeoutExecuted(ctx, packet)
 }
+
+// UnmarshalPacketData tries to extract an application type from raw channel packet data.
+func (k Keeper) UnmarshalPacketData(ctx sdk.Context, rawData []byte) (types.PacketDataI, error) {
```

Comment:
Why doesn't this unmarshal directly into `FungibleTokenPacketData`?

Reply:
Will do.

Reply:
>Why doesn't this unmarshal directly into FungibleTokenPacketData?

yeah there's no need for this function to exist if you unmarshal directly to that type

Reply:
TIL about types.ModuleCdc!